services/network/network.go

```diff
@@ -24,6 +24,14 @@ type SubResource struct {
 	ID *string `json:"id,omitempty"`
 }
 
+// GroupSubResource referes to another subresource, and the group that subresource belongs to
+type GroupSubResource struct {
+	// ID - Resource ID.
+	ID *string `json:"id,omitempty"`
```

Comment:
>ID [](http://example.com/codeflow?start=1&length=2)

in azure a resource id is supposed to be fully qualified, for eg; /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/resourceType/{resourceName}

as far as i understand, we try to keep moc-sdk similar to azure. that means we dont need a new groupsubresource. 

any reason to diverge from this? @msftpgayam

Reply:
discussed offline - I believe the plan is to keep this as is